src/test/java/seedu/address/logic/commands/ListCommandTest.java

```diff
@@ -68,4 +70,33 @@ public void execute_listIsFiltered_showsVideos() {
         String expectedString = String.format(ListCommand.MESSAGE_SUCCESS_VIDEOS, moduleCode, lectureName);
         assertCommandSuccess(listCommand, model, expectedString, expectedModel);
     }
+
+    @Test
+    public void execute_listIsFiltered_moduleNotFound() {
+        ModuleCode moduleCode = new ModuleCode("CS1231S");
```

Comment:
Could you use `VALID_MODULE_CODE_2103` or `VALID_MODULE_CODE_2040` from `CommandTestUtil` instead of `"CS1231S"`?